floris/tools/optimization/scipy/yaw_wind_rose.py

```diff
@@ -265,25 +296,75 @@ def _optimize(self):
         Returns:
             opt_yaw_angles (np.array): Optimal yaw angles of each turbine.
         """
+        opt_yaw_angles = np.array(self.x0, dtype=float)
```

Comment:
flagging this as another place where, to me, baseline yaw angles makes more sense than x0.